tsconfig.json

```diff
@@ -28,7 +32,8 @@
     "config",
     "__tests__",
     "__fixtures__",
-    "noImplictAnyImportBypass.d.ts"
+    "noImplictAnyImportBypass.d.ts",
+    "codegen.ts"
```

Comment:
Originally we added `"./node_modules/@types/node/globals.d.ts"` for globals, but I think line 22 does it better.

I've tested it by typing `console` in a file and the typescript language server doesn't try to import it. The code completion seems to be working, too.